tools/oeedger8r/CMakeLists.txt

```diff
@@ -37,13 +36,23 @@ add_custom_command(
     DEPENDS ${OEEDGER8R_SRC}
 )
 
+if (WIN32)
 add_custom_command(
     OUTPUT ${BINARY}
-    COMMAND ${BUILD_COMMAND}
+    COMMAND C:\\windows\\system32\\WindowsPowerShell\\v1.0\\powershell -command ${CMAKE_CURRENT_SOURCE_DIR}/build.ps1 ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} 
     COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/src/_build/main.native ${BINARY}
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/src
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/src
```

Comment:
You need to put a dependency on the build.ps1 script so that the command is re-run if the script changes. It can eitehr be here or in the OEEDGER8R_SRC variable.

Reply:
OK